### ISSUE TITLE
Is gwann syntax only for testing data?

### ISSUE THREAD

**Issue by SoviaIndrawati** [jhagenauer/gwann#16]
Hello Hagenauer,
I would like to ask if gwann syntax is currently only used for data testing?
Then couldn't Gwann display all the predictions for each location?

I thank you in advance.

**Comment by jhagenauer** [comment]
No, you can use GWANN for prediction. The example usage was a bit confusing. I renamed the variables so it is clearer.

Best
Julian

**Comment by SoviaIndrawati** [comment]
can gwann display all predictions in each location?
